examples/pysdl2.py

```diff
@@ -0,0 +1,357 @@
+                        "character": keycode,
+                        "unmodified_character": keycode,
+                        "modifiers": cef.EVENTFLAG_NONE
+                    }
+                    browser.SendKeyEvent(key_event)
+            elif event.type == sdl2.SDL_KEYUP:
+                # Handle key up events for non-text keys
+                if event.key.keysym.sym in [
+                        sdl2.SDLK_RETURN,
+                        sdl2.SDLK_BACKSPACE,
+                        sdl2.SDLK_DELETE,
+                        sdl2.SDLK_LEFT,
+                        sdl2.SDLK_RIGHT,
+                        sdl2.SDLK_UP,
+                        sdl2.SDLK_DOWN,
+                        sdl2.SDLK_HOME,
+                        sdl2.SDLK_END
+                ]:
+                    keycode = getKeyCode(event.key.keysym.sym)
+                    key_event = {
+                        "type": cef.KEYEVENT_KEYUP,
+                        "windows_key_code": keycode,
+                        "character": keycode,
+                        "unmodified_character": keycode,
+                        "modifiers": cef.EVENTFLAG_NONE
+                    }
+                    browser.SendKeyEvent(key_event)
+        # Clear the renderer
+        sdl2.SDL_SetRenderDrawColor(
+            renderer,
+            backgroundColour.r,
+            backgroundColour.g,
+            backgroundColour.b,
+            255
+        )
+        sdl2.SDL_RenderClear(renderer)
+        # Tell CEF to update which will trigger the OnPaint
+        # method of the RenderHandler instance
+        cef.MessageLoopWork()
+        # Update display
+        sdl2.SDL_RenderCopy(
+            renderer,
+            renderHandler.texture,
+            None,
+            sdl2.SDL_Rect(0, headerHeight, browserWidth, browserHeight)
+        )
+        sdl2.SDL_RenderPresent(renderer)
+    # User exited
+    exit_app()
+    
+def getKeyCode(key):
+    """Helper function to convert SDL2 key codes to cef ones"""
+    if key == sdl2.SDLK_RETURN:
+        return 13
+    if key == sdl2.SDLK_DELETE:
+        return 46
+    if key == sdl2.SDLK_BACKSPACE:
+        return 8
+    if key == sdl2.SDLK_LEFT:
+        return 37
+    if key == sdl2.SDLK_RIGHT:
+        return 39
+    if key == sdl2.SDLK_UP:
+        return 38
+    if key == sdl2.SDLK_DOWN:
+        return 40
+    if key == sdl2.SDLK_HOME:
+        return 36
+    if key == sdl2.SDLK_END:
+        return 35
+    raise Exception("Invalid key")
+
+class LoadHandler(object):
+    """Simple handler for loading URLs."""
+    
+    def OnLoadingStateChange(self, browser, is_loading, **_):
+        if not is_loading:
+            print("loading complete")
+            
+    def OnLoadError(self, browser, frame, error_code, failed_url, **_):
+        if not frame.IsMain():
+            return
+        print("Failed to load %s" % failed_url)
+        cef.PostTask(cef.TID_UI, exit_app, browser)
```

Comment:
Looks like an error here, wrong number of args. Got this exception:
```
Failed to load http://www.speedtest.net/pl/
[CEF Python] ExceptHook: catched exception, will shutdown CEF

Traceback (most recent call last):
  File "task.pyx", line 57, in cefpython_py27.PyTaskRunnable (cefpython_py27.cpp:33940)
TypeError: exit_app() takes no arguments (1 given)
```

When displaying "Failed to load %s" it would be nice to display also error_code. I currently have no idea what happened while loading that speedtest.pl website. Is it really necessary to exit app here on error? I would just display a message that url failed and an error code for a reason why it failed. A website may send 404 error, but it will display some webpage and it should be displayed in browser and not exited.